src/plugins/data_source_management/public/components/direct_query_data_sources_components/connection_detail/direct_query_connection_detail.tsx

```diff
@@ -469,45 +491,48 @@ export const DirectQueryDataConnectionDetail: React.FC<DirectQueryDataConnection
   const tabs = [...conditionalTabs, ...genericTabs];
 
   return (
-    <EuiPage paddingSize="none">
-      <EuiPageBody>
-        <EuiPageHeader style={{ justifyContent: 'spaceBetween' }}>
-          <EuiPageHeaderSection style={{ width: '100%', justifyContent: 'space-between' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem grow={false}>
-                {!useNewUX && (
-                  <EuiText data-test-subj="datasourceTitle" size="s">
-                    <h1>{datasourceDetails.name}</h1>
-                  </EuiText>
-                )}
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPageHeaderSection>
-        </EuiPageHeader>
-        <DatasourceOverview />
-        <EuiSpacer />
-        {integrationsFlyout}
-        {datasourceDetails.status !== 'ACTIVE' ? (
-          <InactiveDataConnectionCallout
-            datasourceDetails={datasourceDetails}
-            fetchSelectedDatasource={fetchSelectedDatasource}
-          />
-        ) : (
-          <>
-            <EuiAccordion
-              id="queryOrAccelerateAccordion"
-              buttonContent={'Get started'}
-              initialIsOpen={true}
-              paddingSize="m"
-            >
-              <QueryOrAccelerateData />
-            </EuiAccordion>
-            <EuiTabbedContent tabs={tabs} size="s" />
-          </>
-        )}
-
-        <EuiSpacer />
-      </EuiPageBody>
-    </EuiPage>
+    <>
+      <EuiPage paddingSize="none">
+        <EuiPageBody>
+          <EuiPageHeader style={{ justifyContent: 'spaceBetween' }}>
+            <EuiPageHeaderSection style={{ width: '100%', justifyContent: 'space-between' }}>
+              <EuiFlexGroup alignItems="center">
+                <EuiFlexItem>
+                  {!useNewUX && (
+                    <EuiText data-test-subj="datasourceTitle" size="s">
+                      <h1>{datasourceDetails.name}</h1>
+                    </EuiText>
+                  )}
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>{dataSourceMenuView}</EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiPageHeaderSection>
+          </EuiPageHeader>
+          <DatasourceOverview />
+          <EuiSpacer />
+          {integrationsFlyout}
+          {datasourceDetails.status !== 'ACTIVE' ? (
+            <InactiveDataConnectionCallout
+              datasourceDetails={datasourceDetails}
+              fetchSelectedDatasource={fetchSelectedDatasource}
+            />
+          ) : (
+            <>
+              <EuiAccordion
+                id="queryOrAccelerateAccordion"
+                buttonContent={'Get started'}
+                initialIsOpen={true}
+                paddingSize="m"
+              >
+                <QueryOrAccelerateData />
+              </EuiAccordion>
+              <EuiTabbedContent tabs={tabs} size="s" />
+            </>
+          )}
+
+          <EuiSpacer />
+        </EuiPageBody>
+      </EuiPage>
+    </>
```

Comment:
why this `<></>` change?

Reply:
Thanks good catch. Had used it earlier while testing out the action menu on top. Later had to move the datasource picker in the flex group to accommodate old nav. So this is not needed anymore.